single_kernel_mongo/managers/mongodb_operator.py

```diff
@@ -1254,3 +1248,18 @@ def clear_password_management_statuses(self) -> None:
                 scope="app",
                 component=self.name,
             )
+
+    def _should_skip_because_of_tls(self) -> bool:
```

Comment:
1. Missing docstring
2. I would perhaps clarify the name, that it's skipping due to an incomplete TLS setup